compiler/gentree.cpp

```diff
@@ -477,6 +477,21 @@ VertexPtr GenTree::get_expr_top(bool was_arrow) {
       break;
     }
     case tok_varg: {
+      // cases:
+      // [...$a]:        prev: tok_opbrk, next: tok_var_name
+      // [1, ...$a]:     prev: tok_comma, next: tok_var_name
+      // [1, ...$a, 1]:  prev: tok_comma, next: tok_var_name
+      // [1, ...f(), 1]: prev: tok_comma, next: tok_func_name
```

Comment:
I suppose, that `array(...$a)` is also a valid syntax?

Reply:
Yep, added.